README.md

```diff
@@ -297,6 +297,12 @@ See the [docs](docs/Custom_Command_Keybindings.md)
 
 ![Interactive Rebasing](../assets/rebase.gif)
 
+### Git flow support
+
+[Git Flow](https://github.com/nvie/gitflow) is a Git branching model proposed by Vincent Driessen that streamlines and standardizes the process of development, release, and maintenance. 
+It introduces a strict branching structure with specific roles: master or main for production-ready code, develop for active development, and feature/*, release/*, hotfix/* branches for features, releases, and bug fixes, respectively. 
+To learn more, refer to Vincent Driessen's original [post](https://nvie.com/posts/a-successful-git-branching-model/).
+
```

Comment:
I'd mention that Lazygit supports git flow, but only if you already have git flow installed

Reply:
That makes sense, I will add that